content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,175 @@
+- **--no-wait**: by default, the command waits a certain amount of time for the events (tests, static analysis violations) to arrive to Datadog.
+This step is important as it makes sure that the events are queryable in Datadog before the rules are executed,
+avoiding incorrect evaluation. If, in your pipeline, the job containing the `datadog-ci gate evaluate` command is
+called several minutes after the related events are sent to Datadog, you could skip this waiting time by specifying the `--no-wait` flag.
+Note that, if used incorrectly, this flag might result in inaccurate rule evaluations.
+
+[Custom scopes](#custom-scope) can be added by using the following option one or more times: **--scope**:
+
+{{< code-block lang="shell" >}}
+datadog-ci gate evaluate --scope team:backend --scope team:frontend
+{{< /code-block >}}
+
+The command logs can be examined to understand what was the overall gate evaluation status, along with information
+about all the rules that were evaluated.
+
+{{< img src="ci/datadog_ci_gate_evaluate_logs.png" alt="Datadog-ci gate evaluate logs" style="width:90%;">}}
+
+## Rule scope
+
+When creating a rule, you can define its scope, which states when it should be evaluated.
+The rule scope is then checked based on the context passed by the `datadog-ci gate evaluate` command to understand whether the rule should be evaluated or not.
+
+For each scope name (for example, "branch"), you can either select included or excluded values.
+When included values are selected, the rule is evaluated if one or more included values are part of the command context.
+When excluded values are selected, the rule is evaluated if none of the excluded values are part of the command context.
+
+For example, to create a rule that is evaluated in all branches but `main` of the `example-repository` repository, you can define the following scope:
+
+{{< img src="ci/scope_not_main_example_repository.png" alt="Rule scope for example-repository and not main branch" style="width:90%;">}}
+
+If a rule does not contain a scope name, it is evaluated for all values of that scope name.
+For example, if a rule does not contain the `repository` scope, it is evaluated for all repositories. If the rule scope is
+set to `always evaluate`, the rule is evaluated on all repositories and branches.
+
+{{< img src="ci/rule_scope_always_evaluate.png" alt="Rule scope for rules always evaluated" style="width:90%;">}}
+
+### Custom scope
+
+In addition to branch and repository, you can define custom scope names to further filter rules that are evaluated for a specific CI pipeline.
+
+Custom scopes can be added when creating a rule in the following way:
+1. Click on **+ Add Filter** and select **Custom Scope**
+2. Define the scope name, for example, `team`.
+3. Define the scope included or excluded values.
+
+Differently from "branch" and "repository" scopes, custom scopes need to be provided in the `datadog-ci gate evaluate` command using the **--scope** option.
```

Comment:
```suggestion
Unliked the `branch` and `repository` scopes, custom scopes need to be provided in the `datadog-ci gate evaluate` command using the **--scope** option.
```

Reply:
I have changed "Unliked" to "Unlike". I guess that's what you meant?